src/main/java/com/exasol/releasedroid/report/ReportFormatterImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.exasol.releasedroid.report;
+
+import static com.exasol.releasedroid.usecases.ReleaseDroidConstants.LINE_SEPARATOR;
+
+import com.exasol.releasedroid.usecases.report.*;
+
+/**
+ * Implementation of {@link ReportFormatter}.
+ *
+ */
+public class ReportFormatterImpl implements ReportFormatter {
+
+    @Override
+    public String formatReport(final Report report) {
+        final StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Maybe we don't need StringBuilder in this and the next method. It will be shorter and no much performance difference